applications/ShapeOptimizationApplication/python_scripts/mesh_controller_with_solver.py

```diff
@@ -38,10 +38,10 @@ def __init__(self, MeshSolverSettings, model):
                     "time_step"       : 1.0
                 },
                 "domain_size"     : 3,
-                "linear_solver_settings" : {
+                "mesh_motion_linear_solver_settings" : {
                     "solver_type" : "AMGCL",
                     "smoother_type":"ilu0",
-                    "krylov_type": "gmres",
+                    "krylov_type": "bicgstab",
```

Comment:
Is there a specific reason for this change?

Reply:
No. Don't know, why this changed. I reset it.